hypervisor/arch/x86/guest/vcpu.c

```diff
@@ -632,6 +632,8 @@ int32_t run_vcpu(struct acrn_vcpu *vcpu)
 		rip = vcpu_get_rip(vcpu);
 		exec_vmwrite(VMX_GUEST_RIP, ((rip+(uint64_t)instlen) &
 				0xFFFFFFFFFFFFFFFFUL));
+
+		exec_vmwrite32(VMX_PROC_VM_EXEC_CONTROLS, vcpu->arch.proc_vm_exec_ctrls);
```

Comment:
Why we need this? We should already initialized proc_vm_exec_ctrls .

Reply:
Now, It is changed by nmi_window and mtf. So we need to cache it when vmexit and write back before vmentry.

Reply:
No. This is too expansive.  Whenever we update the VMCS field, we update the cache register too. This way they are synced all the time